runtime/legion/runtime.cc

```diff
@@ -18003,6 +18049,9 @@ namespace Legion {
           if (!strcmp(argv[i],"-hl:safe_mapper"))
             unsafe_mapper = false;
           BOOL_ARG("-hl:inorder",program_order_execution);
+          BOOL_ARG("-hl:dump_physical_traces",dump_physical_traces);
+          BOOL_ARG("-hl:no_tracing",no_tracing);
+          BOOL_ARG("-hl:no_physical_tracing",no_physical_tracing);
```

Comment:
I don't think you need to add -hl versions of these flags. The old ones are just there for backwards compatibility.

Reply:
I'll remove them.